pkg/apis/config/v1alpha4/types.go

```diff
@@ -118,6 +118,10 @@ type Node struct {
 	// binded to a host Port
 	ExtraPortMappings []PortMapping `yaml:"extraPortMappings,omitempty" json:"extraPortMappings,omitempty"`
 
+	// GPUs allows to access GPU devices from the kind node. Setting this to
+	// "all" will pass all the available GPUs to the kind node.
+	Gpus string `yaml:"gpus,omitempty" json:"gpus,omitempty"`
```

Comment:
use GPUs?
>  All letters in the acronym should have the same case, using the appropriate case for the situation

https://github.com/zecke/Kubernetes/blob/master/docs/devel/api-conventions.md#naming-conventions

Reply:
+1, also this field should be validated (with "all" being the only valid value currently), and we should put a note that in the future we'll look at supporting specifying specific devices.

Reply:
Commented more below and on the issue.

Reply:
done